src/yayson/presenter.coffee

```diff
@@ -69,6 +69,7 @@ module.exports = (utils, adapter) ->
             relationships[key].links = buildLinks links[key]
         else
           relationships[key]= build data
+        delete relationships[key] unless relationships[key].length > 0
```

Comment:
Unfortunately, .length doesn't work on objects, you have to use Object.keys(relationships[key]).length